sa/sa_test.go

```diff
@@ -1592,8 +1567,13 @@ func TestRevokeCertificate(t *testing.T) {
 	// Add a cert to the DB to test with.
 	certDER, err := ioutil.ReadFile("www.eff.org.der")
 	test.AssertNotError(t, err, "Couldn't read example cert DER")
-	issued := sa.clk.Now()
-	_, err = sa.AddCertificate(ctx, certDER, reg.ID, nil, &issued)
+	issued := sa.clk.Now().UnixNano()
+	_, err = sa.AddPrecertificate(ctx, &sapb.AddCertificateRequest{
+		Der:    certDER,
+		RegID:  &reg.ID,
+		Ocsp:   nil,
+		Issued: &issued,
+	})
```

Comment:
I think you'll have to make a similar diff to this in some of the `ocsp-updater` unit tests. E.g. `TestGenerateAndStoreOCSPResponse` from `cmd/ocsp-updater/main_test.go` is failing right now with:
```
--- FAIL: TestGenerateAndStoreOCSPResponse (0.19s)
    main_test.go:105: Couldn't generate OCSP response: sql: no rows in result set
```

Reply:
yorp 😞.

Reply:
Fixed.